README.md

```diff
@@ -19,217 +19,172 @@
-Available Commands:
-  network-policy  Generate network policies based on recorded network activity
-  seccomp-profile Generate seccomp profiles based on recorded syscalls activity
-
-...
-$ kubectl gadget audit --help
-Audit a subsystem
-
-Usage:
-  kubectl-gadget audit [command]
-
-Available Commands:
-  seccomp     Audit syscalls according to the seccomp profile
-
-...
-$ kubectl gadget profile --help
-Profile different subsystems
-
-Usage:
-  kubectl-gadget profile [command]
-
-Available Commands:
-  block-io    Analyze block I/O performance through a latency distribution
-  cpu         Analyze CPU performance by sampling stack traces
-  tcprtt      Analyze TCP connections through an Round-Trip Time (RTT) distribution
-
-...
-$ kubectl gadget snapshot --help
-Take a snapshot of a subsystem and print it
-
-Usage:
-  kubectl-gadget snapshot [command]
-
-Available Commands:
-  process     Gather information about running processes
-  socket      Gather information about TCP and UDP sockets
-
-...
-$ kubectl gadget top --help
-Gather, sort and periodically report events according to a given criteria
-
-Usage:
-  kubectl-gadget top [command]
-
-Available Commands:
-  block-io    Periodically report block device I/O activity
-  ebpf        Periodically report ebpf runtime stats
-  file        Periodically report read/write activity by file
-  tcp         Periodically report TCP activity
-
-...
-$ kubectl gadget trace --help
-Trace and print system events
-
-Usage:
-  kubectl-gadget trace [command]
-
-Available Commands:
-  bind         Trace socket bindings
-  capabilities Trace security capability checks
-  dns          Trace DNS requests
-  exec         Trace new processes
-  fsslower     Trace open, read, write and fsync operations slower than a threshold
-  mount        Trace mount and umount system calls
-  network      Trace network streams
-  oomkill      Trace when OOM killer is triggered and kills a process
-  open         Trace open system calls
-  signal       Trace signals received by processes
-  sni          Trace Server Name Indication (SNI) from TLS requests
-  tcp          Trace TCP connect, accept and close
-  tcpconnect   Trace connect system calls
-  tcpdrop      Trace TCP kernel-dropped packets/segments
-  tcpretrans   Trace TCP retransmissions
-
-...
+gadgetctl --remote-address=tcp://$IP:1234
+gadgetctl run trace_open
 ```
 
-## How does it work?
+***The above demonstrates the simplest command. To learn how to filter, export, etc. please consult the documentation for the [run](/docs/guides/run.md) command***.
+ 
+## Core concepts
+
+### What is a Gadget?
+
+Gadgets are the central component in the Inspektor Gadget framework. A Gadget is an [OCI image](https://opencontainers.org/) that includes one or more eBPF programs, metadata YAML file and, optionally, WASM modules for post processing and a logo file.
+As OCI images, they use the same tooling as containers and share the same attributes; shareable, modular, deployable, etc.
```

Comment:
What do you mean with tooling? You mean you can use `oras`, `crane`, etc. to manipulate them? I think it's confusing because then you talk about "deployment", so I don't want users to think that they can run Gadgets with docker 😅. 

```suggestion
As OCI images, they can be stored in a container registry (compliant with the OCI specifications), making them easy to distribute and share.
```